truffle/test/Owned.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const Owned = artifacts.require('Owned');
+const truffleAssert = require('truffle-assertions');
+
+contract('Owned', (accounts) => {
+    let instance;
+
+    // Initiating new contract before each test
+    beforeEach("Creating new contract", async () => {
+        await Owned.new()
+            .then(_instance => {
+                instance = _instance;
+            });
+    });
+
+    // Testing getOwner function.
+    it('Owner should be ' + accounts[0], () => {
+        return instance.getOwner()
+            .then(owner => {
+                assert.strictEqual(owner, accounts[0]);
+            });
+    });
+
+    // Testing changeOwner function.
+    it('Owner should be ' + accounts[1], () => {
+        return instance.changeOwner(accounts[1])
+            .then(() => {
+                return instance.getOwner();
+            })
```

Comment:
```javascript
.then(() => instance.getOwner())
.then(owner =>
```

Reply:
It can indeed been simplified. However, I must admit that I prefer this way. It seems more "explicit" if you know what I mean.

Reply:
Haha. It took me time to get used to it.